Made some minor content fixes (#44)
scaffold-eth/se2-docs: docs/components/EtherInput.md

## Props

| Prop                       | Type       | Default Value | Description                                                                             |
| -------------------------- | ---------- | ------------- | --------------------------------------------------------------------------------------- |
| **value**                  | `string`   | `undefined`   | You can enter ether quantity or USD quantity, but value will always be stored in ETH.   |
| **onChange**               | `function` | `undefined`   | A callback invoked when the amount in the EtherInput changes.                           |

> | **placeholder** (optional) | `string`   | `undefined`   | The string that will be rendered before EtherInput has been entered.                    |

Reviewer: ```suggestion
| **placeholder** (optional) | `string`   | `undefined`   | The string that will be rendered when there is not input value.                    |
```
